sdk/identity/Azure.Identity/CHANGELOG.md

```diff
@@ -1,16 +1,21 @@
 # Release History
-## 1.3.0-preview.1 (Unreleased)
+## 1.3.0-beta.1 (2020-09-11)
 
 ### New Features
 - Restoring Application Authentication APIs from 1.2.0-preview.6
+- Added respect of `CancellationToken` to the `MsalPublicClient` and `MsalConfidentialClient` ([#13201](https://github.com/Azure/azure-sdk-for-net/issues/13201))
+- Moved to the latest App Services MSI API version ([#13687](https://github.com/Azure/azure-sdk-for-net/issues/13687))
 - Added `IncludeX5CClaimHeader` to `ClientCertificateCredentialOptions` to enable subject name / issuer authentication with the `ClientCertificateCredential`.
 - Added `RedirectUri` to `InteractiveBrowserCredentialOptions` to enable authentication with user specified application with a custom redirect url.
 - Added `IdentityModelFactory` to enable constructing models from the Azure.Identity library for mocking.
+- Unify exception handling between `DefaultAzureCredential` and `ChainedTokenCredential` ([#14408](https://github.com/Azure/azure-sdk-for-net/issues/14408))
 
 ### Fixes and improvements
+- Fixed `VisualStudioCodeCredential` crashes on macOS (Issue [#14362](https://github.com/Azure/azure-sdk-for-net/issues/14362))
 - Fixed issue with non GUID Client Ids (Issue [#14585](https://github.com/Azure/azure-sdk-for-net/issues/14585))
 - Update `VisualStudioCredential` and `VisualStudioCodeCredential` to throw `CredentialUnavailableException` for ADFS tenant (Issue [#14639](https://github.com/Azure/azure-sdk-for-net/issues/14639))
-
+- Fixed issue with `DefaultAzureCredential` incorrectly catching `AuthenticationFailedException` (Issue [#14974](https://github.com/Azure/azure-sdk-for-net/issues/14974))
+- Fixed issue with `DefaultAzureCredential` throwing exceptions during concurrent calls (Issue [#15013](https://github.com/Azure/azure-sdk-for-net/issues/15013))
 
 ## 1.2.2 (2020-08-20)
 
```

Comment:
You need to add the notes from 1.2.3 release here as well.

Reply:
Added notes